apps/cloudinary/src/index.js

```diff
@@ -223,6 +238,14 @@ setup({
       "value":"auto,none,gif,webp,bmp,flif,heif,heic,ico,jpg,jpe,jpeg,jp2,wdp,jxr,hdp,png,psd,arw,cr2,svg,tga,tif,tiff",
       "required": true,
       "default": "auto"
+    },
+    {
+      "id": "defaultTransformations",
+      "name": "Default Transformations",
+      "description": "Specific upload transformations. When this is specified, 'Media Quality' and 'Format' are ignored. See https://cloudinary.com/documentation/transformations_on_upload for available transformations. Needs to be a valid JSON array, e.g. '[{\"quality\": 15, \"width\": 100, \"height\": 100, \"crop\": \"crop\"}]'.",
+      "type": "Symbol",
+      "required": false,
+      "default": "[]"
```

Comment:
A free form field sounds rather risky: it will make this app hard to be backwards compatible, should Cloudinary API change, and requires a non trivial validation to be guarded against malicious injections.

What about adding the transformations you are after as the other items in this list?